crates/types/src/network.rs

```diff
@@ -4,29 +4,27 @@
 // You should have received a copy of the MIT License
 // along with the HotShot repository. If not, see <https://mit-license.org/>.
 
-use std::{
-    env, fs,
-    num::NonZeroUsize,
-    ops::Range,
-    path::{Path, PathBuf},
-    time::Duration,
-    vec,
-};
+use std::{fs, ops::Range, path::Path, time::Duration, vec};
 
-use clap::ValueEnum;
-use hotshot_types::{
+use crate::hotshot_config_file::HotShotConfigFile;
+use crate::{
     constants::REQUEST_DATA_DELAY, light_client::StateVerKey, traits::signature_key::SignatureKey,
-    ExecutionType, HotShotConfig, PeerConfig, ValidatorConfig,
+    HotShotConfig, ValidatorConfig,
 };
+use clap::ValueEnum;
 use libp2p::{Multiaddr, PeerId};
 use serde_inline_default::serde_inline_default;
-use surf_disco::Url;
 use thiserror::Error;
-use toml;
-use tracing::{error, info};
-use vec1::Vec1;
+use tracing::error;
 
-use crate::client::OrchestratorClient;
+/// default number of rounds to run
+pub const ORCHESTRATOR_DEFAULT_NUM_ROUNDS: usize = 100;
+/// default number of transactions per round
+pub const ORCHESTRATOR_DEFAULT_TRANSACTIONS_PER_ROUND: usize = 10;
+/// default size of transactions
+pub const ORCHESTRATOR_DEFAULT_TRANSACTION_SIZE: usize = 100;
+/// default delay before beginning consensus
+pub const ORCHESTRATOR_DEFAULT_START_DELAY_SECONDS: u64 = 60;
```

Comment:
can these live in `types/src/constants.rs`?

Reply:
Done